sdk/identity/identity/rollup.base.config.js

```diff
@@ -33,7 +33,7 @@ export function nodeConfig(test = false) {
 
   if (test) {
     // entry point is every test file
-    baseConfig.input = ["dist-esm/test/public/*.spec.js", "dist-esm/test/internal/**/*.spec.js"];
+    baseConfig.input = ["dist-esm/test/internal/*.spec.js", "dist-esm/test/public/*.any.spec.js", "dist-esm/test/public/*.node.spec.js"];
```

Comment:
What is the guideline we are using for the "any.spec.ts" files?

Reply:
This is the first time I see the need for separating tests based on what's possible to import.

The public API of the browser and the node version of these credentials don't expose the same types.

If they exposed the same types, I would skip the tests, but I can't, since the missing classes break the build if I leave the files on the browser.

We could fix this in many ways. Some I can think of are:

- This one, in which we separate the test files based on the names of the files.
- To separate the test files based on folders instead of file names.
- To make the browser exports consistent with the node exports.

What approach would be better?

Reply:
What we decide will be included in the guide for sure.

Reply:
According to Will's latest test guidelines work, this should be moved to use folders inside of internal and public, like `internal/**/node/*.spec.ts`. I'll do so!